pkgs/applications/altcoins/default.nix

```diff
@@ -34,4 +34,9 @@ rec {
 
   stellar-core = callPackage ./stellar-core.nix { };
 
+  zcash = callPackage ./zcash {
+    withGui = false;
+    openssl = pkgs.openssl_1_1_0;
+    boost = pkgs.boost163;
```

Comment:
Right, I changed these 2 lines without documenting them. One sec, fixing this to pass it from the param.